htdocs/install/mysql/data/llx_c_action_trigger.sql

```diff
@@ -109,6 +109,7 @@ insert into llx_c_action_trigger (code,label,description,elementtype,rang) value
 insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('MEMBER_EXCLUDE','Member excluded','Executed when a member is excluded','member',27);
 insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('FICHINTER_VALIDATE','Intervention validated','Executed when a intervention is validated','ficheinter',30);
 insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('FICHINTER_MODIFY','Intervention modify','Executed when a intervention is modify','ficheinter',30);
+insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('FICHINTER_SIGN','Intervention signed','Executed when a intervention is signed','ficheinter',31);
```

Comment:
Ok to add the "signed" column

However, for trigger, we must try to reduce number of triggers to CRUD methods only.
So signing will be setting a flag on intervention so updating the intervention. The trigger FICHINTER_MODIFY (the U of CRUD) should be used for that. 
We may add fill the $object->context to show we are in a signing process.